jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -209,9 +228,13 @@ spec:
             memory: 1Gi
       serviceAccountName: jupyterhub-hub
       volumes:
-      - configMap:
-          defaultMode: 420
-          name: jupyterhub-cfg
-        name: config
+      - name: config 
+        configMap:
+            defaultMode: 420
+            name: jupyterhub-cfg 
+      - name: sidecar-script
+        configMap:
+          defaultMode: 511
```

Comment:
Just one tiny comment, any specific reason this is mode 511?

Reply:
Yeah, this one is because by default there is no execution permission for the script.